src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -680,7 +680,8 @@ public void testCacheAndCheckWhichAddsNewResourceLocationButKeepsSameCheckerInst
 
         final DynamicalResourceHolderCheck check = new DynamicalResourceHolderCheck();
         final String firstExternalResourceLocation = getPath("checks" + File.separator
-            + "imports" + File.separator + "import-control_one.xml");
+            + "imports" + File.separator + "importcontrol"
+            + File.separator + "InputImportControlOne.xml");
```

Comment:
please copy that ml file to folder for Checker input files.
Each module should have its own input files, nothing is sharable.